source/languages/en/riakcs/cookbooks/configuration/Configuring-Riak-CS.md

```diff
@@ -336,28 +349,36 @@ data.riakcs.net
 
 ## Garbage Collection Settings
 
-The following options are available to make adjustments to the Riak CS
-garbage collection system. More details about garbage collection in
-Riak CS are available in [[Garbage Collection]].
-
-* `leeway_seconds` --- The number of seconds that must
-  elapse before an object version that has been explicitly deleted or
-  overwritten is eligible for garbage collection. The default value is
-  86400 (24 hours).
-* `gc_interval` --- The interval, in seconds, at which the garbage
-  collection daemon runs to search for and reap eligible object
-  versions. The default value is 900 seconds (15 minutes). It is
-  important that you have only _one_ garbage collection daemon running
-  in a cluster at any point in time. To disable the daemon on a node,
-  set the `gc_interval` parameter to `infinity`.
-* `gc_retry_interval` --- The number of seconds that must elapse
-  before another attempt is made to write a record for an object
-  manifest in the `pending_delete` state to the garbage collection
-  eligibility bucket. In general, this condition should be rare, but
-  could happen if an error condition caused the original record in the
-  garbage collection eligibility bucket to be removed prior to the
-  reaping process completing. The default value is 21600 seconds (6
-  hours).
+The following options are available to make adjustments to the Riak CS garbage
+collection system. More details about garbage collection in Riak CS are
+available in [[Garbage Collection]].
+
+* `gc.leeway_period` (`leeway_seconds` in `advanced.config`) --- The amount of
```

Comment:
..."in `advanced.config` or `app.config`" for all of these (or drop `advanced.config` in favor of `app.config` since if they're using `advanced.config` it'd be easier to put the setting in `riak.conf`)